src/mame/drivers/nmk16.cpp

```diff
@@ -3964,11 +3964,11 @@ TIMER_DEVICE_CALLBACK_MEMBER(nmk16_state::nmk16_scanline)
 MACHINE_CONFIG_START(nmk16_state::tharrier)
 
 	/* basic machine hardware */
-	MCFG_DEVICE_ADD("maincpu", M68000, 10000000) /* 10 MHz */
+	MCFG_DEVICE_ADD("maincpu", M68000, 10_MHz_XTAL) // TMP68000P-12, 10 MHz
```

Comment:
Isn't TMP68000P-12 rated for 12 MHZ?

Reply:
That's fine, it's common to underclock(or even overclock) chips